resources/migrations/003-create-user-reviews.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE user_reviews (
+       id INTEGER PRIMARY KEY,
+       from_uid TEXT NOT NULL,
+       to_uid TEXT NOT NULL,
+       review_id TEXT NOT NULL,
+       FOREIGN KEY (from_uid) REFERENCES USERS(id),
+       FOREIGN KEY (to_uid) REFERENCES USERS(id),
+       FOREIGN KEY (review_id) REFERENCES REVIEWS(id)
```

Comment:
Id is integer in Reviews table and text here. Should we normalize and use text only for all the different ids in different tables.